gammapy/estimators/utils.py

```diff
@@ -346,3 +347,55 @@ def compute_lightcurve_fvar(lightcurve, flux_quantity="flux"):
     )
 
     return table
+
+
+def compute_lightcurve_dtime(lightcurve, flux_quantity="flux"):
+    r"""Compute the doubling time of the input lightcurve.
+
+    Internally calls the `~gammapy.stats.compute_dtime` function
+
+
+    Parameters
+    ----------
+    lightcurve : '~gammapy.estimators.FluxPoints'
+        the lightcurve object
+    flux_quantity : str
+        flux quantity to use for calculation. Should be 'dnde', 'flux', 'e2dnde' or 'eflux'. Default is 'flux'.
+        Useful in case of custom lightcurves computed outside gammapy
+
+    Returns
+    -------
+    table : `~astropy.table.Table`
+        Table of doubling times and associated error for each energy bin of the lightcurve.
+    """
+
+    flux = getattr(lightcurve, flux_quantity)
+    flux_err = getattr(lightcurve, flux_quantity + "_err")
+    time = lightcurve.geom.axes["time"].center
+
+    time_id = flux.geom.axes.index_data("time")
+
+    dtime, dtime_err = compute_dtime(flux.data, flux_err.data, time, axis=time_id)
+
+    energies = lightcurve.geom.axes["energy"].edges
+    table = Table(
+        [
+            energies[:-1],
+            energies[1:],
+            dtime.T[0],
+            dtime_err.T[0],
+            np.abs(dtime.T[1]),
+            dtime_err.T[1],
+        ],
+        names=(
+            "min_energy",
+            "max_energy",
+            "doubling_time",
+            "dtime_err",
+            "halving_time",
+            "htime_err",
+        ),
+        meta=lightcurve.meta,
```

Comment:
I don't know if this meta info is relevant for the table...

Reply:
Absolutely, it is mostly a placeholder for future metadata or to not leave the table without